z3/src/ast.rs

```diff
@@ -1525,6 +1528,26 @@ impl<'ctx> Array<'ctx> {
             })
         }
     }
+
+    /// Returns true if the array is a const array(a.is_const_array() <=> exists v, forall i. select(a, i) == v)
```

Comment:
Is this correct?

Reply:
That is coming from https://github.com/prove-rs/z3.rs/blob/3c9766336297769467be0ddcbce0376b58a344f3/z3-sys/src/lib.rs#L520-L522. So I believe so? Maybe only a single implication now that I think of it because I'm not sure if a nested store gets collapsed. (store (store ((as const (Array Int Int)) 9) 0 1281) 0 9)

Reply:
Sorry, I had commented while doing something else and from my phone away from home.

My concern was actually about the formatting and if that should be in the summary line at the top of the comment.

Should also have a blank line before the example header.